pipelines/mnist/mnist_pipeline_on_prem.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import kfp.dsl as dsl
+from kubernetes import client as k8s_client
+
+
+def mnist_train_op(tf_data_dir: str, tf_model_dir: str,
+                   tf_export_dir: str, train_steps: int, batch_size: int,
+                   learning_rate: float, step_name='Training'):
+    return dsl.ContainerOp(
+        name=step_name,
+        image='docker.io/jinchi/mnist_model:0.2',
```

Comment:
Shouldn't this be TRAINING_IMAGE_NAME?

I'm not too sure if it's actually useful to use a parameter here though. I'd say we should just use an image at gcr.io/kubeflow-examples/mnist/model instead of a custom one, so people can jump into using the pipeline easier

Reply:
You are absolutely right. I think that's better to public image 
gcr.io/kubeflow-examples/mnist/model, but I not sure if that's updated after some code merged.